tests/kernel/install_ltp.pm

```diff
@@ -222,12 +222,18 @@ sub add_ltp_repo {
 }
 
 sub get_default_pkg {
-    if (is_sle) {
-        return 'qa_test_ltp qa_test_ltp-32bit' if (is_released);
-        return 'ltp'                           if (is_jeos);
+    my @packages;
```

Comment:
I'd use plain string, just remove `qa_test_ltp-32bit`. And if you want to use array, there is `@pkgs` array in other methods, having consistency would be nice (no matter which variable name wins).  But no hard feeling about it, can be merged this way.

Reply:
I'll leave it as is.